docs/docs/components/view.md

```diff
@@ -62,7 +62,7 @@ ignorePointerEvents: boolean = false; // web only
 // setFocusRestricted() below.
 // WARNING: For the sake of performance, this property is readonly and
 // changing it during the View life cycle will produce an error.
-restrictFocusWithin: boolean = false; // web only
+restrictFocusWithin?: boolean = false; // web only
```

Comment:
We don't use "?" within the documentation. The fact that it has an "= false" on the end means that it's optional and that "false" is the default value.

Reply:
Is this really "web only"? Or does it apply to all keyboard-based systems? If it's the latter, delete the "// web only" comment.